docs/ATLAS.md

```diff
@@ -0,0 +1,23 @@
+### Setup Atlas using terraform ###
+
+Follow these steps to setup a M2 cluster in Atlas to be used by the uDeploy portal.
+
+1. Register an API key in Atlas.  `Atlas -> Project -> Access management -> API Keys`
+    
+    - Click `Manage -> Create API Key`
+    - Enter a short description for this key
+    - Give the key `Project Owner` permission (This permission will be reduced at the end of these steps)
+    - Click `Next`
+    - Copy the private key and store it in a secure place (This will be the only time this key will be visible and available to be copied)
+    - API Whitelist: It is recommended to add just your current network external IP for security reasons. 
+    - Click `Done`
+    - Copy the public key and store it in a secure place
```

Comment:
"Save the public key for later use" might suffice since it is up to the user on how they save it.